docs/apis/webadmin.md

```
@@ -364,3 +364,47 @@ Revokes administrator rights of user `user1@linagora.com` for domain `linagora.c
 - `204` if successful, even if the user exists but was not an admin (idempotent).
 - `404` if the domain or user does not exist.
 - `400` if `domainName` or `username` has an invalid format.  
+
+---
+
+### Running the resource rights reposition task
+
+This task allows re-applying or correcting CalDAV resource rights based on the current resources administrators stored
+in the database.
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)